NEWS.md

```diff
@@ -32,6 +34,9 @@
 * in `leftjoin`, `rightjoin`, and `outerjoin` the `indicator` keyword argument
   is deprecated in favor of `source` keyword argument; `indicator` will be removed
   in 2.0 release ([2649](https://github.com/JuliaData/DataFrames.jl/pull/2649))
+* applying `map` to `GroupedDataFrame` is now deprecated. The return value
+  of this function might change in 2.0 release
+  ([#2662](https://github.com/JuliaData/DataFrames.jl/pull/2662))
```

Comment:
Here again, it would be annoying to have to wait for 2.0 if we want to change this. How about backporting it to 0.22?

Reply:
OK. I will change this PR in a way to allow it to be included in 0.22.6 release.